examples/tools/go_db.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import pyscf
+from pyscf.tools.go_db import get_xyz
+import requests
```

Comment:
We can get it to work, by downloading the database.txt file and putting the path to the file in the script. 
However, I feel it is ugly and everyone would have to edit the script to put their respective paths.
I feel this is ugly. The current approach is cleaner because people can just run the script without making any changes to the script. 

However, if you prefer the file approach I can fix that.

Reply:
When you clone PySCF, you get a database of basis sets (STO-3G, 6-31G, cc-pVDZ, def2-TZVPP, etc.) for each element in the periodic table, downloaded. Then PySCF can just read that database file directly. So the part about this being "ugly" because of needing to edit the script with a personalized "path", is not necessary.

The disadvantage of having the database shipped with PySCF is that it's huge. I won't try to include a database of this size in PySCF without prior approval (and perhaps some studies on what would be the best compression/decompression method, if any).

So for now, the URL approach is best.